airflow/providers/telegram/provider.yaml

```diff
@@ -0,0 +1,40 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+---
+package-name: apache-airflow-providers-telegram
+name: Telegram
+description: |
+    `Telegram <https://telegram.org/>`__
+
+versions:
+  - 0.0.1
+
+integrations:
+  - integration-name: Telegram
+    external-doc-url: https://telegram.org/
```

Comment:
See: https://github.com/apache/airflow/blob/eacf40d85eb014e5a0350d46a3fe68625d8d5f15/airflow/provider.yaml.schema.json#L35-L41

Reply:
Got it. Thank you.